ssl/ssl_test.cc

```diff
@@ -2962,6 +2962,31 @@ class SSLVersionTest : public ::testing::TestWithParam<VersionParam> {
     return GetParam().ssl_method == VersionParam::is_dtls;
   }
 
+  void CheckCounterInit() {
+    EXPECT_EQ(SSL_CTX_sess_connect(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_connect(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_connect_good(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_connect_good(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_connect_renegotiate(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_connect_renegotiate(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept_renegotiate(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept_renegotiate(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept_good(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_accept_good(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_hits(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_hits(server_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_cb_hits(client_ctx_.get()), 0);
+    EXPECT_EQ(SSL_CTX_sess_cb_hits(server_ctx_.get()), 0);
```

Comment:
We're missing a test when SSL_CTX_sess_cb_hits is not 0.

Reply:
Added a test called `SessionExternalCacheHit`